ios/Sources/TimetableFeature/TimetableContent.swift

```diff
@@ -31,14 +32,35 @@ public struct TimetableContent: View {
         self.store = store
     }
 
+    private let dateFormatter: DateFormatter = {
+       let formatter = DateFormatter()
+        formatter.setLocalizedDateFormatFromTemplate("HH:mm")
+        return formatter
+    }()
```

Comment:
Is it better to create global static value for dateformatter?
I also used in https://github.com/DroidKaigi/conference-app-2021/pull/637 as well.

Reply:
do we need timezone?

Reply:
These `.first()` not needed 😭 (It's my mistake)
I'll remove this.

Reply:
> Is it better to create global static value for dateformatter?

Yes, it is. I think so too.

> do we need timezone?

maybe. but currently it is not clear. (we need to check API response)